lib/MusicBrainz/Server/Controller/WS/2/Tag.pm

```diff
@@ -66,9 +66,16 @@ sub tag_submit : Private
     $self->_validate_post($c);
 
     my $xp = MusicBrainz::Server::WebService::XML::XPath->new( xml => $c->request->body );
+    my @nodelist;
+    eval {
+        @nodelist = $xp->find('/mb:metadata/*/*')->get_nodelist;
+    };
+    if ($@) {
```

Comment:
I'm a bit confused, what does this mean here? :)

Reply:
Just one of those special perl vars you have to memorize.  If there's an error inside the `eval`, Perl sets `$@` to the error: https://perldoc.perl.org/perlvar#%24%40

Reply:
So it's effectively a try-catch, ok :)